pkg/kubelet/util/util_test.go

```diff
@@ -149,3 +151,71 @@ func TestIsUnixDomainSocket(t *testing.T) {
 		assert.Equal(t, result, test.expectSocket, "Unexpected result from IsUnixDomainSocket: %v for %s", result, test.label)
 	}
 }
+
+func TestGetContainerByIndex(t *testing.T) {
+	type res struct {
+		container v1.Container
+		ok        bool
+	}
+	tests := []struct {
+		title      string
+		containers []v1.Container
+		statuses   []v1.ContainerStatus
+		idx        int
+		res        res
```

Comment:
As this test only compares 'name' and 'ok' values, would it be better to have them in the test case structure, e.g.:
```suggestion
		expectedName: string
		expectedOK: bool
```
This would make test cases more readable and eliminate the need in a separate 'res' structure.

Reply:
Thinks, is it ok if I change it to the following?

expectedContainer v1.Container 
expectedOK        bool